packages/plasma-b2c/src/components/Tokens/Tokens.stories.tsx

```diff
@@ -68,11 +68,11 @@ const StoryDemo = ({ context }) => {
 
     return (
         <>
-            {Object.entries(groupedTokens).map(([category, value]) => (
+            {Object.entries(groupedTokens).map(([category, subcategoties]) => (
```

Comment:
__minor__

Опечатка

```suggestion
            {Object.entries(groupedTokens).map(([category, subcategories]) => (
```